indico/modules/rb_new/client/js/modules/bookRoom/BookRoomModal.jsx

```diff
@@ -397,3 +400,55 @@ export default class BookRoomModal extends React.Component {
         );
     }
 }
+
+const mapStateToProps = (state, {roomId}) => {
+    const {
+        bookRoom: {filters: {recurrence, dates, timeSlot}, bookingForm: {timeline}},
+        user,
+    } = state;
+    const room = roomsSelectors.getRoom(state, {roomId});
+    return {
+        bookingData: {recurrence, dates, timeSlot},
+        availability: timeline && timeline.availability,
+        dateRange: timeline && timeline.dateRange,
+        user,
+        room,
+    };
+};
+
+const mapDispatchToProps = (dispatch) => ({
+    onSubmit: (
+        {reason, usage, user, isPrebooking},
+        {bookingData: {recurrence, dates, timeSlot}, room: roomData}) => {
```

Comment:
`room: roomData` do we even have to rename the `room` to `roomData`? It is used a few lines below in the same fashion which cancels the rename.

Reply:
Good one, thanks!